### PR TITLE
Take into account the new layout of parametrics

### PR DESCRIPTION
**Pull request by lthms**
The layout of Tezos protocol parametrics have changed.

We need to double check this is correct before merging I guess.

### CODE REVIEW

File: mondaynet/values.yaml [hunk 1]
```diff
@@ -126,25 +126,33 @@ activation:
...
+      enable: true
+      origination_size: 6314
+      challenge_window_in_blocks: 40
+      max_available_messages: 1000000
+      stake_amount: "32000000"
+      commitment_period_in_blocks: 20
+      max_lookahead_in_blocks: 30000
+      max_active_outbox_levels: 20160
+      max_outbox_messages_per_level: 100
+      number_of_sections_in_dissection: 32
```
Comment (nicolasochem) [inline mondaynet/values.yaml:155]: mondaynet starts from Jakarta and updates to alpha so it's probably not needed yet?

File: dailynet/values.yaml [hunk 2]
```diff
@@ -103,37 +103,39 @@ activation:
...
+      finality_period: 10
+      max_inboxes_count: 15
+      withdraw_period: 10
+      max_messages_per_inbox: 1010
+      max_commitments_count: 30
+      cost_per_byte_ema_factor: 120
+      max_withdrawals_per_batch: 15
+      max_ticket_payload_size: 2048
+      rejection_max_proof_size: 30000
+      sunset_level: 2760
```
Comment (nicolasochem) [inline dailynet/values.yaml:121]: I assume dailynet will break starting today unless we merge this?

**Comment by lthms** [comment]
So I have been told that this MR might not be the right move, since the encoding have been made backward compatible. So the old JSON schema is still the correct one.

**Comment by nicolasochem** [comment]
> So I have been told that this MR might not be the right move, since the encoding have been made backward compatible. So the old JSON schema is still the correct one.

Why would we stay on the old format? It still makes sense to move to the new encoding on dailynet? But without any change in values.

Mondaynet starts from jakarta so it's not applicable there yet.

On the other hand, I have already increased the value of TORU sunset level so next mondaynet won't have the problem we talked about on slack.

**Comment by lthms** [comment]
The OCaml types have been updated, but the new encodings have been implemented so that they are compatible with the previous ones.

That is, the new OCaml types are an implementation details as far as the rest of the word is concerned.

**Comment by nicolasochem** [comment]
great, closing in this case.

**Pull request closed without merging**